lemur/certificates/service.py

```diff
@@ -194,7 +194,6 @@ def get_all_certs_not_attached_to_endpoint_with_autorotate():
         .filter(Certificate.rotation == true())
         .filter(Certificate.revoked == false())
         .filter(Certificate.not_after >= arrow.now())
-        .filter(not_(Certificate.replaced.any()))
```

Comment:
Why remove this?

Reply:
This should be an improvement as it's confusing that we don't turn it off after it's already reissued. Also we still maintain audibility of manual vs automated reissuance via the description and replacement relation